modules/apps/commerce/commerce-wish-list-service/src/main/java/com/liferay/commerce/wish/list/service/impl/CommerceWishListLocalServiceImpl.java

```diff
@@ -348,7 +347,4 @@ protected void validateGuestWishLists() throws PortalException {
 	@Reference
 	private CommerceWishListItemLocalService _commerceWishListItemLocalService;
 
-	@Reference
-	private DDMFormValuesHelper _ddmFormValuesHelper;
```

Comment:
@JanisZhang Could you look into the git history to see why this was added? If it was added with usages, but then was forgotten when the usages were removed, then it's safe to remove this.

Reply:
Hi @dantewang , `_ddmFormValuesHelper` was replaced by `Objects.equals  ` in 23ec8cfe9df7ea0c8e727c277264dcb686d54b28, it's safe to delete. Thanks!